src/module.cpp

```diff
@@ -187,6 +187,10 @@ void module::assign(const module& m)
             {
                 copy_ins = add_return(copy_inputs);
             }
+            else if(ins->name() == "@divzero")
```

Comment:
Do we need to add the same check in the `insert_generic_instructions` ?

Reply:
Yeah I think so. Just adding this in and re-running tests.